docker/build_docker.sh

```diff
@@ -249,6 +249,43 @@ function docker_build {
             return 1
         fi
     fi
+
+    echo "================= $(date): Starting version verifocation on image: ${image_name} ================="
+    echo "Checking that python version is match to the base version..."
```

Comment:
Maybe we should use more clear wording, not really clear from this sentence what we are going  to do